### DISCUSSION TITLE
Braw files won’t show in library after drag and dropped from Braw tool box

### DISCUSSION THREAD

**Discussion by Sallen307** [latenitefilms/BRAWToolbox#145] [General]
After I process my Braw files in Braw tool box and Drag and Drop into my library only red audio clips are shown. It appears that the video file is missing but I do have audio. The Braw clip is visible in Braw tool box but doesn't fully convert in FCPX Library .I haven't seen this issue on the thread has anyone else experienced this issue.![image](https://github.com/latenitefilms/BRAWToolbox/assets/139857037/7202e372-9df9-4785-a54d-9a1e3596dafa)

**Comment by latenitefilms** [comment]
Solved in #144.
